solutions/standard/variables.tf

```diff
@@ -82,3 +90,41 @@ variable "cbr_rules" {
   default     = []
   # Validation happens in the rule module
 }
+
+########################################################################################################################
+# KMS variables
+########################################################################################################################
+
+variable "existing_kms_instance_crn" {
+  type        = string
+  default     = "crn:v1:bluemix:public:kms:us-south:a/abac0df06b644a9cabc6e44f55b3880e:d4559998-17ab-4ab8-ba9e-a0c9d2c4ef23::"
```

Comment:
should we pass the default value of existing kms crn?

Reply:
oops...that was by mistake. Removed now.